getting-started-guides/csp/gcp/spark-gpu/rapids.sh

```diff
@@ -24,16 +29,10 @@ if [[ "${ROLE}" != 'Master' ]]; then
   # Ensure we have GPU drivers installed.
   "${RAPIDS_INIT_ACTION_DIR}/internal/install-gpu-driver.sh"
 else
-  readonly STORAGE_BUCKET=my-bucket
-  gsutil cp gs://${STORAGE_BUCKET}/xgboost4j-spark_2.11-1.0.0-Beta2.jar /usr/lib/spark/python/lib/
-  gsutil cp gs://${STORAGE_BUCKET}/xgboost4j-spark_2.11-1.0.0-Beta2.jar /usr/lib/spark/jars/
-  gsutil cp gs://${STORAGE_BUCKET}/xgboost4j_2.11-1.0.0-Beta2.jar /usr/lib/spark/jars/
-  gsutil cp gs://${STORAGE_BUCKET}/cudf-0.9.1-cuda10.jar /usr/lib/spark/jars/
-
-  wget "https://rapidsai-data.s3.us-east-2.amazonaws.com/spark/mortgage.zip"
-  unzip "mortgage.zip"
-  hadoop fs -mkdir "/tmp/xgboost4j_spark"
-  hadoop fs -copyFromLocal mortgage "/tmp/xgboost4j_spark/mortgage"
+  gsutil cp ${GCS_BUCKET}/xgboost4j-spark_2.11-1.0.0-Beta2.jar /usr/lib/spark/python/lib/
+  gsutil cp ${GCS_BUCKET}/xgboost4j-spark_2.11-1.0.0-Beta2.jar /usr/lib/spark/jars/
+  gsutil cp ${GCS_BUCKET}/xgboost4j_2.11-1.0.0-Beta2.jar /usr/lib/spark/jars/
```

Comment:
Beta3

Reply:
done, also need to change maven url and from `xgboost4j_2.11` to `xgboost4j_2.x`